### DISCUSSION TITLE
p.251 JWT import 에러 문제

### DISCUSSION THREAD

**Discussion by DEC-DEV** [fsoftwareengineer/todo-application#130] [질문과 답변] [answered]
![error1](https://user-images.githubusercontent.com/103486552/191430860-3b5b6f10-639c-4ec4-a84c-e296b747259a.PNG)
사진처럼 jwt 관련해서 아예 import가 불가능 합니다.

![error2](https://user-images.githubusercontent.com/103486552/191430879-a6514909-3095-4329-a98a-bc061831601a.PNG)
gradle 7.5버전이라 그런가 싶어서 검색해봤는데 다른 소스에서 10.11.5로 진행하길래 그걸로도 해봤는데 impor가 안됩니다.
bulid path에 넣으면 된다는 소리 듣고 demo/build/libs 파일에 넣어봤는데도 import가 안됩니다.

![error3](https://user-images.githubusercontent.com/103486552/191430883-81c035cd-8477-4514-b8e2-656325d8ed48.PNG)
cmd 들어가서 gradlew build --refresh-dependencies 로 빌드 클린해보고 
빌드 캐시 삭제해보고 마지막으로 위 사진처럼 디펜던시 확인했는데
디펜던시는 잘 인스톨 된거 같은데 진행이 안되고 있습니다.

해결법 아시는 분 있으시면 도움 부탁드립니다.

**Comment by DEC-DEV** [comment, marked as answer]
간단한 건데 어렵게 돌아갔네요.
https://veneas.tistory.com/entry/Eclipse-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8%EC%97%90-jar-%ED%8C%8C%EC%9D%BC-%EC%B6%94%EA%B0%80%ED%95%98%EB%8A%94-%EB%B0%A9%EB%B2%95-%EC%9E%90%EB%B0%94-%EB%9D%BC%EC%9D%B4%EB%B8%8C%EB%9F%AC%EB%A6%AC-%EC%B6%94%EA%B0%80
위 방법으로 해결했습니다.

**Comment by holic158** [comment]
gradle 버전이 다를 경우 아래와 같이 포함해보세요

// https://mvnrepository.com/artifact/io.jsonwebtoken/jjwt
implementation group: 'io.jsonwebtoken', name: 'jjwt', version: '0.9.1'
